wagtail/images/formats.py

```diff
@@ -15,30 +15,29 @@ def __init__(self, name, label, classnames, filter_spec):
 
     def editor_attributes(self, image, alt_text):
         """
-        Return string of additional attributes to go on the HTML element
+        Return additional attributes to go on the HTML element
         when outputting this image within a rich text editor field
         """
-        return 'data-embedtype="image" data-id="%d" data-format="%s" data-alt="%s" ' % (
-            image.id, self.name, escape(alt_text)
-        )
+        return {
+            'data-embedtype': "image",
+            'data-id': image.id,
+            'data-format': self.name,
+            'data-alt': escape(alt_text),
+        }
 
     def image_to_editor_html(self, image, alt_text):
         return self.image_to_html(
             image, alt_text, self.editor_attributes(image, alt_text)
         )
 
-    def image_to_html(self, image, alt_text, extra_attributes=''):
+    def image_to_html(self, image, alt_text, extra_attributes={}):
```

Comment:
Please don't use [mutable default arguments](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments). Do this instead:
```python
def img_tag(self, extra_attributes=None):
    if extra_attributes is None:
        extra_attributes = {}
```